desafio-14/lucasfturos/csharp/Program.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.IO;
+using System.Globalization;
+using System.Collections.Generic;
+
+namespace ExpressoesNumericas
+{
+    class Program
+    {
+        static double AvaliaExpressao(string expressao)
+        {
+            expressao = expressao.Replace(" ", "");

```

Comment:
Considere conferir primeiro se a string é vazia ou null antes de fazer este replace. Se uma string nula for repassada ao método ele falhará nesta linha e a próxima checagem se torna inútil.